common.yaml

```diff
@@ -48,7 +48,8 @@ components:
       type: object
       allOf:
         - $ref: "#/components/schemas/api-response-common"
-        - properties:
+        - type: object
```

Comment:
shouldn't this be `- $ref: "#/components/schemas/result_info"` instead of introducing another level of modelling? [`result_info`](https://github.com/cloudflare/api-schemas/blob/main/common.yaml#L27-L45) looks to have the correct structure.

Reply:
alternatively, let's do away with the `$ref`s entirely and make this explicit and one less level of indirection to trace.

```yaml
api-response-collection:
  type: object
  required:
    - success
    - errors
    - messages
  properties:
    result_info:
      $ref: "#/components/schemas/result_info"
    success:
      description: Whether the API call was successful.
      type: boolean
      enum:
        - true
      example: true
    errors:
      $ref: "#/components/schemas/messages"
    messages:
      $ref: "#/components/schemas/messages"
```